src/main/scala/vsys/blockchain/state/opcdiffs/SystemTransferDiff.scala

```diff
@@ -63,7 +63,7 @@ object SystemTransferDiff extends OpcDiffer {
       senderDiff = OpcDiff(relatedAddress = senderRelatedAddress, portfolios = senderPortDiff)
       senderTotalDiff = OpcDiff.opcDiffMonoid.combine(senderCallDiff, senderDiff)
       recipientCallDiff <- getTriggerCallOpcDiff(context, senderTotalDiff, sender, recipient, amount, sysTokenId, CallType.Trigger, 1)
-      recipientRelatedAddress = if (recipient.dataType == DataType.Address) Map(Address.fromBytes(recipient.data).explicitGet() -> true) else Map[Address, Boolean]()
+      recipientRelatedAddress = if (recipient.dataType == DataType.Address) Map(Account.fromBytes(recipient.data,0).explicitGet()._1 -> true) else Map[Account, Boolean]()
```

Comment:
I think we can use Address directly here

Reply:
or use `recipientRelatedAddress = Map[Account, Boolean](recipientAddr._1 -> true)` directly

Reply:
ok